cheatsheets_draft/Sensitive_Data_Storage_Cheat_Sheet.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
+
+
+This Cheat Sheet provides guidance on the various areas that need to be considered related to storing sensitive data.
+Since sensitive data storage is not a new concept but instead can be referrenced by compbining aspects of other things, this cheatsheet links to the relevant parts of other cheatsheets.
+
+# Contents
+
+- [Storage Systems](#system-sec)
+    - [DBMS-Security](#dbms-sec)
+    - [Filesystem storage](#fs-sec)
+    - [Cloud and Bucket Storage](#cloud-fs)
+- [Managing secrets](#pain-and-suffering)
+    - [Managing Keys and Salts](#kms)
+- [Bonus Considerations](#data-storage-security)
+    - [Encrypting data per user](#encryption-per-user)
+    - [Tokenizing](#tokenizing)
+- [Access to Data and Permissions](#permissions-and-access)
+
+# Storage Systems
+
+## DBMS Security
+Often sensitive data is stored in databases which need to be secured, in order to do so, please refer to the [database security cheatsheet|https://github.com/owasp/CheatSheetSeries/blob/master/cheatsheets/Database_Security_Cheat_Sheet.md]
+
+## File System Security
+If a filesystem is used to store Sensitive Data, then please refer to the [file storage location |https://github.com/owasp/CheatSheetSeries/blob/toc/cheatsheets/File_Upload_Cheat_Sheet.md#file-storage-location], [user permissions| https://github.com/owasp/CheatSheetSeries/blob/toc/cheatsheets/File_Upload_Cheat_Sheet.md#user-permissions] and [file system permissions | https://github.com/northdpole/CheatSheetSeries/blob/toc/cheatsheets/File_Upload_Cheat_Sheet.md#filesystem-permissions]
+
+
+## Cloud Storage
+<there isn't a relevant cheatsheet for this>
```

Comment:
how it will differ than DBMS Security or File System Security? What do you mean by Cloud Storage? For me Cloud Storage is a vague term to describe all methods how you can store files in the cloud e.g. DBMS in cloud or File System in cloud. 
There is clearly more storage systems in the cloud so mabey we should be more specific here? because for DBMS in cloud we will use the same proptections as for on-prem DBMS

Reply:
meant s3 and similar. but you're right it is essentially file storage